opentargets.json

```diff
@@ -810,8 +801,8 @@
         "literature": {
           "$ref": "#/definitions/literature"
         },
-        "pathwayName": {
-          "$ref": "#/definitions/pathwayName"
+        "pathways": {
```

Comment:
In this case, can pathway.id be null or should we map the gene set name to a reactome ID?

Reply:
So, I kept the pathway name and pathway id definitions as they were. Either of them can be missing, but none can be `null`. I saw that the definition of the gene set name was a bit confusing allowing both pathway name from reactome and also gene set name. But I didn't change the definition.

Reply:
If it can be missing it's fine, thanks. I was thinking more about `pathway.id` with can be a gene set ID. When reviewing the current evidence, I don't know what this field corresponds to. If it doesn't exist we can remove the reference to the gene set ID in the description but if it does we should change/remove the pattern for this field. What do you think?

Reply:
For me the `gene_set` rather sounds like `pathway.name` rather than `pathway.id`. 

However in the current schema `pathwayId` looks like this:

```json
    "pathwayId": {
      "type": "string",
      "description": "Pathway or gene set ID",
      "pattern": "^(R-HSA-)\\d+$",
      "examples": [
        "R-HSA-4615885"
      ]
    },
```

Reply:
For me too. Then I would suggest to remove the gene set ID reference in pathwayId:
```
    "pathwayId": {
      "type": "string",
      "description": "Pathway ID",
      "pattern": "^(R-HSA-)\\d+$",
      "examples": [
        "R-HSA-4615885"
      ]
    },
```